lib/haml_lint/options.rb

```diff
@@ -21,7 +21,7 @@ def parse(args)
       end.parse!(args)
 
       # Any remaining arguments are assumed to be files
-      @options[:files] = args
+      @options[:files] = args.empty? ? ["."] : args
```

Comment:
I have it, but why should that make any difference? the args should be empty anyway and it will inspect just 0 files.